src/internal/views/nunjucks/view-licences/expired-licence.njk

```diff
@@ -22,43 +22,48 @@
         <p>Registered to <a href="/user/{{primaryUser.userId}}/status">{{primaryUser.userName}}</a></p>
       {% endif %}
 
-      {% set returnsHtml %}
-        {% include "nunjucks/view-licences/tabs/returns.njk" %}
-      {% endset %}
+      {% if (returns.length > 0 or communications.length > 0 or (showChargeVersions and chargeVersions.length > 0)) %}
+        <div class="govuk-tabs" data-module="tabs">
+          <h2 class="govuk-tabs__title">Contents</h2>
+          <ul class="govuk-tabs__list">
+            {% if returns.length > 0 %}
+              <li class="govuk-tabs__list-item">
```

Comment:
Does the govuk-tabs component automatically select the first link in the tab navigation?
The HTML for the component shows `govuk-tabs__list-item--selected` as the class on the first element

Reply:
The govuk component does set the selected tab.